src/Sylius/Behat/Page/Shop/Checkout/CompletePageInterface.php

```diff
@@ -20,71 +20,29 @@
 
 interface CompletePageInterface extends SymfonyPageInterface
 {
-    /**
-     * @param string $productName
-     * @param string $quantity
-     *
-     * @return bool
-     */
-    public function hasItemWithProductAndQuantity($productName, $quantity);
-
-    /**
-     * @return bool
-     */
-    public function hasShippingAddress(AddressInterface $address);
-
-    /**
-     * @return bool
-     */
-    public function hasBillingAddress(AddressInterface $address);
-
-    /**
-     * @return bool
-     */
-    public function getPaymentMethodName();
-
-    /**
-     * @return bool
-     */
-    public function hasPaymentMethod();
-
-    /**
-     * @return bool
-     */
-    public function hasShippingMethod(ShippingMethodInterface $shippingMethod);
-
-    /**
-     * @param float $amount
-     *
-     * @return bool
-     */
-    public function hasProductDiscountedUnitPriceBy(ProductInterface $product, $amount);
-
-    /**
-     * @param float $total
-     *
-     * @return bool
-     */
-    public function hasOrderTotal($total);
-
-    /**
-     * @param string $notes
-     */
-    public function addNotes($notes);
-
-    /**
-     * @param string $promotionTotal
-     *
-     * @return bool
-     */
-    public function hasPromotionTotal($promotionTotal);
-
-    /**
-     * @param string $promotionName
-     *
-     * @return bool
-     */
-    public function hasPromotion($promotionName);
+    public function hasItemWithProductAndQuantity(string $productName, string $quantity): bool;
+
+    public function hasShippingAddress(AddressInterface $address): bool;
+
+    public function hasBillingAddress(AddressInterface $address): bool;
+
+    public function getPaymentMethodName(): string;
+
+    public function hasPaymentMethod(): bool;
+
+    public function hasProductDiscountedUnitPriceBy(ProductInterface $product, float $amount): bool;
+
+    public function hasOrderTotal(float $total): bool;
+
+    public function getBaseCurrencyOrderTotal(): string;
+
+    public function hasShippingMethod(ShippingMethodInterface $shippingMethod): bool;
+
+    public function addNotes(string $notes): void;
+
+    public function hasPromotionTotal(string $promotionTotal): bool;
+
+    public function hasPromotion(string $promotionName): bool;
```

Comment:
maybe you should put it in alphabetical order?

Reply:
IMO it is unnecessary, there is no coding standard telling about that